src/main/java/com/minecolonies/coremod/entity/ai/util/RecipeStorage.java

```diff
@@ -65,6 +114,7 @@ public RecipeStorage(final List<ItemStack> input, final int gridSize, final Item
 
     /**
      * Get the list of output items.
+     *
      * @return the copy of the list.
      */
     @SuppressWarnings("squid:S2384")
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Suppressing the 'squid:S2384' warning is not allowed [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1309)